Lib/platform.py

```diff
@@ -824,6 +825,7 @@ def from_subprocess():
                 ['uname', '-p'],
                 stderr=subprocess.DEVNULL,
                 text=True,
+                encoding="locale",
```

Comment:
I think `uname` output is not localized.
https://github.com/coreutils/coreutils/blob/8d4768c94d0fa9de545a6e1c370f9a6fae4cb3a7/src/uname.c#L314-L371

So we can use "utf-8" or "latin1" here.